appveyor.yml

```diff
@@ -37,6 +37,7 @@ build: off
 matrix:
 #  fast_finish: true
   allow_failures:
+    - nodejs_version: "4"
```

Comment:
I see that Node 6 tests seem to pass on AppVeyor, it's just Node 4 that's problematic. Maybe we could disable allowing failures in Node 6?